RHIDP-12595: Migrate RHDH UI features to New Frontend System (#2107)
redhat-developer/red-hat-developers-documentation-rhdh: modules/shared/proc-control-what-you-see-first-on-the-homepage.adoc

:_mod-docs-content-type: PROCEDURE

[id="control-what-you-see-first-on-the-homepage_{context}"]
= Control what you see first on the homepage

[role="_abstract"]

> Choose which templates and widgets you see when you log in by configuring the dynamic homepage plugin. You can use customizable layouts where you drag and resize cards, or fixed layouts with predefined positions.

Reviewer: ```suggestion
Configure the dynamic homepage plugin to choose which templates and widgets appear when you log in. You can use customizable layouts to drag and resize cards, or fixed layouts with predefined positions.
```
